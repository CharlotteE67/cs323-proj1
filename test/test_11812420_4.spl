struct test{

}

int test_func_1(int a, ){
    return a!=0;
}


int main(){
    int i = 10;
    int b = i - 1;
    int c = test_func_1(a);
    return 0;
