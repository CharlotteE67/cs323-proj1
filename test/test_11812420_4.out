Error type B at Line 4: Missing semicolon ';'
Error type B at Line 4: Unexpected comma ','
Error type B at Line 17: Missing right curly braces '}'
